packages/edit-site/src/components/posts-app/posts-list.js

```diff
@@ -1,7 +1,16 @@
+/**
```

Comment:
Unrelated with this PR but this file should be moved to its own components/post-list folder or something, it's not specific to "posts app"